### PR TITLE
Added definition of ready

### PR DESCRIPTION
**Pull request by rcknight**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,17 @@ The team may need to consult the user to determine whether the idea adds value.
 
 #### Definition of Ready
 
+A Trello card can be considered ready to be worked on if it meets the criteria defined by the INVEST acronym:
+
+- I: Independent (of other stories)
+- N: Negotiable (does not prescribe exactly one solution)
+- V: Valuable (on its own)
+- E: Estimable (approximately - using the process defined in the [Estimating Effort](#estimating-effort) section)
+- S: Small (so that it fits within a sprint)
```
Comment (oscarhirst) [inline README.md]: Maybe something about it being as small as possible, while still being valuable.

Comment (oscarhirst) [reply README.md]: This is mentioned in the story writing section, so possibly not necessary.

Comment (rcknight) [reply README.md]: I think that is a story writing concern, rather than an "is it ready to work on" concern really?

Comment (rcknight) [reply README.md]: I think i'll rewrite this and steal it from the scrum book though..

**Pull request merged**